packages/engine/Source/Shaders/EllipsoidVS.glsl

```diff
@@ -13,8 +13,8 @@ void main()
     vec4 p = vec4(u_radii * position, 1.0);
 
     v_positionEC = (czm_modelView * p).xyz;     // position in eye coordinates
-    gl_Position = czm_modelViewProjection * p;  // position in clip coordinates
-
+    vec4 positionEC = czm_modelView * position;
+    gl_Position = czm_projection * positionEC;
```

Comment:
`positionEC` is not the same as `p`, so this is changing the result.

With the change from the previous comment, this line could be:
```glsl
gl_Position = czm_projection * pEC;
```